process/block/shardblock.go

```diff
@@ -75,6 +76,9 @@ func NewShardProcessor(arguments ArgShardProcessor) (*shardProcessor, error) {
 	}
 
 	pruningDelay := uint32(arguments.Config.StateTriesConfig.UserStatePruningQueueSize * pruningDelayMultiplier)
+	if pruningDelay < defaultPruningDelay {
+		pruningDelay = defaultPruningDelay
```

Comment:
add a log warn as this is a bad configuration

Reply:
Done.